tikv-configuration-file.md

```diff
@@ -271,6 +271,18 @@ Configuration items related to storage
 + Default value: `"2GB"`
 + Unite: MB|GB
 
+### `enable-ttl`
+
++ TTL is short for "Time to live". If this item is enabled, TiKV automatically deletes data that reaches its TTL. To set the value of TTL, you need to specify it in the requests when writing data via the client. If the TTL is not specified, it means that TiKV does not automatically delete the corresponding data.
++ Note: The TTL feature is only available for the RawKV interface for now. You can only configure this feature when creating a new cluster because different interfaces use different data formats in the bottom layer. If you modify this item on an existing cluster, the cluster reports errors when starting.
```

Comment:
```suggestion
+ Note: The TTL feature is only available for the RawKV interface for now. You can only configure this feature when creating a new cluster because TTL uses different data formats in the storage layer. If you modify this item on an existing cluster, TiKV reports errors when starting.
```